subjects/java/checkpoints/double-linkedlist/README.md

```diff
@@ -0,0 +1,135 @@
+## Double Linked List
+
+### Instructions
+
+Create a double linked list data structure that implements the following methods:
+
+- `at(int index)`: to access an element by its index. If the index is out of bound return -1.
+- `add(int value)`: to add an element at the end of the list.
+- `remove(int index)`: to remove an element by its index.
```

Comment:
it already specified in the subject and i added the test for in the last commit

Reply:
I don't see it specified in any place in the subject and it's not shown in the examples provided. Could you tell me where (line, section or paragraph) where the behaviour of calling `remove` with an out-of-bounds index is described?

Reply:
do you mean for the `remove` method or for the `at` method?

Reply:
@zanninso `remove`. And this information is also missing in the `SingleLinkedList` subject that has already been merged. If I had to implement this, I wouldn't know what should happen when `remove` is called with an out-of-bounds index. 

I would probably submit an implementation (praying is the good one), and if not readjust from the trace of the test that is failing (if this is "verbose" enough). I feel that learner should have a clearer idea of what's expected from their solutions instead of using the tests failure as the ONLY source of information for a specific implementation detail.